brunch-config.js

```diff
@@ -28,5 +34,14 @@ module.exports = {
         })
       ]
     }
+  },
+  sourceMaps: 'inline', // For karma-sourcemap-loader.
+  overrides: {
+    production: {
+      paths: {
+        // Exclude files in "test" from compilation.
+        watched: ['app', 'vendor']
```

Comment:
Fair enough. In that case, I have two follow-up questions:

1. What is the practical purpose of having a `vendor` directory in the first place if vendor scripts get packaged with the distributed application anyway?
2. How does `node_modules` relate to `vendor`? I suppose `node_modules` can be treated as a `vendor` directory, except I can neither move nor rename it.